components/gitpod-db/src/typeorm/team-db-impl.ts

```diff
@@ -146,10 +171,18 @@ export class TeamDBImpl implements TeamDB {
             );
         }
 
+        let slug = slugify(name, { lower: true });
         const teamRepo = await this.getTeamRepo();
+
+        const existingTeam = await teamRepo.findOne({ slug, deleted: false, markedDeleted: false });
+        if (!!existingTeam) {
```

Comment:
**question**: Do we want to support slugs based on the org name when it's the first org and skip the suffix? It feels like we're rewarding the users grabbing a namespace first but is this something we want? Cc @atduarte

Reply:
Yes 😁 If we need to take it back later, we will

Reply:
![ok](https://user-images.githubusercontent.com/120486/226892036-b3c8be5c-b726-4c5d-bd38-4420d1319b49.gif)